labs.yml

```diff
@@ -80,3 +80,10 @@ commands:
         description: target catalog to migrate schema to
       - name: to-schema
         description: target schema to migrate tables to
+
+  - name: validate-groups-membership
+    description: Validate the groups to see if the groups at account level and workspace level have different membership
+    table_template: |-
+      Workflow_Group_Name\tAccount_Group_Name
```

Comment:
Use spaces for separating words, not underscores

Reply:
Replaced _ with space